requirements.txt

```diff
@@ -25,6 +25,15 @@ onnxscript==0.1.0.dev20241018
 monai==1.4.0
 gif==23.3.0
 scikit-image==0.24.0
+py-spy==0.4.0
```

Comment:
Could you please check if the packages added are really required ? I doesn't found any import of some of them in the comited code.